templates/event/event_show.html.twig

```diff
@@ -104,7 +104,7 @@
                 {% endif %}
 
                 {% if event.description %}
-                    <p class="p-1">{{event.description}}</p>
+                    <p class="p-1">{{event.description | raw }}</p>
```

Comment:
@daria-sieroshtan 
this can potentially lead to cut tags. I suggest to update "cut" filter to prevent this